### PR TITLE
fix: add error page for expired services

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-5965
| License          | BSD 3-Clause

## Description

Add error page for expired services along with the side-bar

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/expired/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import managerErrorPage from '@ovh-ux/manager-error-page';
+
+import routing from './expired.routing';
+
+const moduleName = 'ovhManagerDedicatedExpired';
+
+angular.module(moduleName, ['ui.router', managerErrorPage]).config(routing);
```
Comment (varun257) [inline packages/manager/apps/dedicated/client/app/expired/index.js:7]: `import '@uirouter/angularjs';`

Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/expired/index.js:8]: This is done. Thanks

**Review by varun257 — DISMISSED** [review]

**Review by varun257 — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/expired/expired.routing.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import get from 'lodash/get';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.expired', {
+    url: '/expired',
+    params: {
+      error: null,
+    },
+    component: 'managerErrorPage',
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/expired/expired.routing.js:9]: I'm having trouble understanding the difference with the error page here

Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/expired/expired.routing.js:9]: This is the same as the error page, but with the sidebar displayed :)

**Review by antleblanc — APPROVED** [review]

**Pull request merged**